src/Emulator/Cores/tcg.cproj

```diff
@@ -6,7 +6,9 @@
     <Architecture Condition=" $(Architecture) == '' ">i386</Architecture>
     <TcgDirectory>tlib/tcg</TcgDirectory>
     <OutputDirectory>$(TcgDirectory)/bin/$(Configuration)</OutputDirectory>
+    <PropertiesLocation>..\..\..\..\..\output\properties.csproj</PropertiesLocation>
```

Comment:
Did you find it required to be added here? We use `$(CompilerPath)` in this file, and we didn't have to import anything.

Reply:
Yes, it must be added.  `$(CompilerPath)` appears to be different as it's a property of the execution unit.  Without this, it doesn't work.  At least not under msbuild.

Reply:
I added a commit passing ArPath as property as well.